### PR TITLE
Keep apps table sorted

### PR DESCRIPTION
**Pull request by thgh**
First of all, thank you for your contribution! 😄

Please make sure you have read [contribution guidelines](https://github.com/caprover/caprover/blob/master/CONTRIBUTING.md#before-contributing). 

### Most important items

- Make sure to communicate your proposed changes ob our Slack channel beforehand.
- Large PRs (50+ lines of code) will get rejected due to increased difficulty for review - unless they have been communicated beforehand with project maintainers.
- **Refactoring work will get rejected** unless it's been communicated with project's maintainers **beforehand**. There is a thousand ways to write the same code. Every time a code is changed, there is a potential for a new bug. We don't want to refactor the code just for the sake of refactoring.

These rules are strictly enforced to make sure that we can maintain the project moving forward.

### CODE REVIEW

File: src/containers/apps/AppsTable.tsx [hunk 1]
```diff
@@ -87,10 +86,11 @@ class AppsTable extends Component<
                 align: ALIGN,
                 sorter: (a, b) => {
                     return (
-                        new Date(b.lastDeployTime).getTime() -
-                        new Date(a.lastDeployTime).getTime()
+                        Date.parse(a.lastDeployTime) -
+                        Date.parse(b.lastDeployTime)
                     )
                 },
+                sortDirections: ['descend', 'ascend'],
```
Comment (thgh) [inline src/containers/apps/AppsTable.tsx:93]: These changes keep the existing behaviour but fix the label when hovering the table header.

**Comment by githubsaturn** [comment]
Thanks!

**Comment by githubsaturn** [comment]
Looks like linter failed, can you please fix and resubmit?

**Comment by githubsaturn** [comment]
Thanks!

**Pull request merged**